### PR TITLE
Matchup Guide: non blue fair decks

### PR DESCRIPTION
**Pull request by Lutri4eva**
article + banner

### CODE REVIEW

File: markdown/articles/2021/08/Non_blue_fair_decks.txt [hunk 1]
```diff
@@ -0,0 +1,115 @@
+---
+authors: OtterLady
+banner: /assets/stinging-study.c21.jpg
+title: “Matchup Guides: Non Blue Fair Decks”
```
Comment (angrybacon) [inline markdown/articles/2021/08/Non_blue_fair_decks.txt:4]:
```suggestion
title: Matchup Guides: Non Blue Fair Decks
```

File: markdown/articles/2021/08/Non_blue_fair_decks.txt [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+---
+
+Gude!
+
+This time i want to talk how our Gameplan against Non Blue Fair Decks looks like and how to execute it.
+All these matchups have one common point, a fast Doomsday is always good and the post Doomsday part of the game is far more interesting.
+
+!EXCLAIMER! I don't give any sideboard guides in these articles. Sideboard guides don't help anyone learning to play a deck, because you first have to know how to play the deck to know how to sideboard correctly.
+
+##DnT
```
Comment (angrybacon) [inline markdown/articles/2021/08/Non_blue_fair_decks.txt:14]: Add a space after `#` in titles (and avoid abbreviations in titles)
```suggestion
## Death and Taxes
```

File: markdown/articles/2021/08/Non_blue_fair_decks.txt [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+###Gameplan
```
Comment (angrybacon) [inline markdown/articles/2021/08/Non_blue_fair_decks.txt:16]: Add blank lines after titles

```suggestion
### Gameplan

```

File: markdown/articles/2021/08/Non_blue_fair_decks.txt [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+###Cards to keep in mind
+####Maindeck
+<row variant="hand"> {{!Kaldra Compleat}} {{!Spirit of the Labyrinth}} {{!Rishadan Port}} {{!Wasteland}} {{!Solitude}}</row>
+
+####Sideboard
+<row variant="hand"> {{!Deafening Silence}} {{!Mindbreak Trap}} {{!Surgical Extraction}</row>
+
+###Gameplay
+####Pre Doomsday
+Against DnT resolving an early {{Doomsday}} is the safest way to win. Mulliganing for a Turn 1 or Turn 2 Doomsday is a good starting point.
```
Comment (angrybacon) [inline markdown/articles/2021/08/Non_blue_fair_decks.txt:28]: Add blank lines after paragraphs. Otherwise, they won't be considered *paragraphs* but part of the same block of text (no new line).

If you actually want a simple carriage return (new line but no spacing), append 2 consecutive spaces at the end of the line in your file.

```suggestion
Against DnT resolving an early {{Doomsday}} is the safest way to win. Mulliganing for a Turn 1 or Turn 2 Doomsday is a good starting point.

```

File: markdown/articles/2021/08/Non_blue_fair_decks.txt [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+Game 1 they have no way to interact with your Doomsday on the stack, so you can just slam as early as possible.
+In post board games play around {{Mindbreak Trap}} when it's reasonable and keep countermagic ready to fight {{Deafening Silence}}.
+
+####Post Doomsday 
+it's very important to keep in mind, that your mana is your biggest weakness and that you can not rely on cyclers as tool to work through your pile.
+There are many possible piles, but one of the easiest, most tested and pretty {{Surgical Extraction}} proof pile is Fetchland, Fetchland, Thassa's Oracle, Land, Land.
+It gives you the necessary mana, thins the pile and goes down to 0 cards to play around {{Solitude}}. All without cyclers or cantrips that get stuck in your hand against a {{Spirit of the Labyrinth}}.
+It's also very customizable. For instance if they already build up some pressure, add a {{Massacre}} or a removal spell for {{Stoneforge Mystic}} to stay safe from {{Kaldra Compleat}}.
+When we are already talking about Kaldra Compleat, remember it has haste and represents 5 damage for 2 Mana out of a Stoneforge Mystic.
+
```
Comment (angrybacon) [inline markdown/articles/2021/08/Non_blue_fair_decks.txt:39]: Reduce consecutive blank lines to 1 maximum

**Review by angrybacon — CHANGES_REQUESTED** [review]

**Comment by angrybacon** [comment]
@Lutri4eva The file should be Markdown and should be nested in a folder named after the date. Moreover, prefer lowercase *kebab-case* over *snake_case*:

```diff
-markdown/articles/2021/08/Non_blue_fair_decks.txt
+markdown/articles/2021/08/18/non-blue-fair-decks.md
```

File: markdown/articles/2021/08/Non_blue_fair_decks.txt (hunk 1 above)
Comment (Lutri4eva) [reply markdown/articles/2021/08/Non_blue_fair_decks.txt]: changed

File: markdown/articles/2021/08/Non_blue_fair_decks.txt (hunk 2 above)
Comment (Lutri4eva) [reply markdown/articles/2021/08/Non_blue_fair_decks.txt]: changed

File: markdown/articles/2021/08/Non_blue_fair_decks.txt (hunk 3 above)
Comment (Lutri4eva) [reply markdown/articles/2021/08/Non_blue_fair_decks.txt]: added blank lines

File: markdown/articles/2021/08/Non_blue_fair_decks.txt (hunk 4 above)
Comment (Lutri4eva) [reply markdown/articles/2021/08/Non_blue_fair_decks.txt]: added space were necessary

**Comment by Lutri4eva** [comment]
Maybe this should rather be a new Meandeck chapter than an article series.
Opinions?

I added all decks in one huge file. We can split them up if necessary.

Jeskai Ragavan, Delver and Bant still need text, but i wanted to get my progress out for you.

If you have any ideas for 1 or 2 sentences for the Gameplan sections, pls message me or comment.

**Comment by angrybacon** [comment]
> Maybe this should rather be a new Meandeck chapter than an article series.
> Opinions?
> 
> I added all decks in one huge file. We can split them up if necessary.

So the difference between the 2 is that chapter are timeless while articles are dated and inherit a context like bans and meta. 

I think matchups deserve a *timeless* resource, but like I mentioned over DMs with Doishy this is not easily feasible currently, or rather not easily consumable by readers: document will be very long and links to specific headings will have to be made by hand by either the author or the reader (`id` attribute in the HTML source).

That's why I'm working on automatic tables of content with links in the next version, as well as collapsible content (think accordion, kind of like decklists currently https://ddft.wiki/appendices/decklists/).  
That way we'll be able to have all of the matchups in the same page which will be easier to maintain and share.

For now depending on the final length I think either could work but I prefer a chapter I think, so that the route already exists and links made today will still work for the next version.

File: markdown/articles/2021/08/22/Non-blue-fair-decks.md [hunk 6]
```diff
@@ -0,0 +1,429 @@
+---
+authors: OtterLady
+banner: /assets/stinging-study.c21.jpg
+title: "Matchup Guides: Non Blue Fair Decks"
+---
+
+With this chapter we want to give you a brief overview over the most relevant matchups in the current meta. We try to keep everything short and easy to read.
+
+!If you want to skip to a certain matchup use STRG+F!
```
Comment (Doishy) [inline markdown/articles/2021/08/22/Non-blue-fair-decks.md:9]: STRG+F, is that like CTRL+F?

File: markdown/articles/2021/08/22/Non-blue-fair-decks.md [hunk 7]
```diff
@@ -0,0 +1,429 @@
...
+
```
Comment (Doishy) [inline markdown/articles/2021/08/22/Non-blue-fair-decks.md:10]: Maybe lead with a description of how each entry is laid out?

eg:

Opposing archetype name:
Gameplan: A short summary of how we should approach the matchup.
Cards to keep in mind: The main relevant cards both pre-boarding and post-boarding in that the opposing deck may have.
Gameplay: What sort of hands you should look to keep and how you should try to play (build depending). This is looked at both before the "combo turn" and after a successful DD resolution.

File: markdown/articles/2021/08/22/Non-blue-fair-decks.md [hunk 8]
```diff
@@ -0,0 +1,429 @@
...
+There are many possible piles, but one of the easiest, most tested and pretty {{Surgical Extraction}} proof pile is Fetchland, Fetchland, Thassa's Oracle, Land, Land.  
+It gives you the necessary mana, thins the pile and goes down to 0 cards to play around {{Solitude}}. All without cyclers or cantrips that get stuck in your hand against a {{Spirit of the Labyrinth}}.  
+It's also very customizable. For instance if they already build up some pressure, add a {{Massacre}} or a removal spell for {{Stoneforge Mystic}} to stay safe from {{Kaldra Compleat}}.  
+When we are already talking about Kaldra Compleat, remember it has haste and represents 5 damage for 2 Mana out of a Stoneforge Mystic.  
+
+## Maverick
+
+### Gameplan
+
+An early Doomsdays gives you enough time to build safe piles.
```
Comment (Doishy) [inline markdown/articles/2021/08/22/Non-blue-fair-decks.md:108]: Doomsdays -> Doomsday

File: markdown/articles/2021/08/22/Non-blue-fair-decks.md [hunk 9]
```diff
@@ -0,0 +1,429 @@
...
+
+#### Sideboard
+
+<row variant="hand"> {{!Deafening Silence}} {{!Mindbreak Trap}} {{!Surgical Extraction}} {{!Choke}} {{!Thoughtseize}}</row>
+
+### Gameplay
+
+#### Pre Doomsday
+
+The matchup against Maverick plays pretty similar to the DnT mactchup, but with a small but important difference, the "I-Win" Button {{Dark Depths}}. Luckily the Dark Depths lines are all very telegraphed and easy to spot.
```
Comment (Doishy) [inline markdown/articles/2021/08/22/Non-blue-fair-decks.md:124]: Are we lumping Mav and GW depths together? I see them as quite different myself.

File: markdown/articles/2021/08/22/Non-blue-fair-decks.md [hunk 10]
```diff
@@ -0,0 +1,429 @@
...
+
+#### Pre Doomsday
+
+This will be a short one. Just cast Doomsday asap and play around possible {{Mindbreak Traps}} in sideboarded games. Like in any Dark Depths deck, their plays to creat a 20/20 are super telegraphed.
+
+#### Post Doomsday 
+
+They only have one card to interact with you, {{Endurance}}. You should always have enough time for a pile that beats Endurance and even plays around Mindbreak Trap.
+
+## GW Depths
```
Comment (Doishy) [inline markdown/articles/2021/08/22/Non-blue-fair-decks.md:193]: `This answers my earlier question. In which case I am not sure that Depths out of reg mav is all that common?

File: markdown/articles/2021/08/22/Non-blue-fair-decks.md [hunk 11]
```diff
@@ -0,0 +1,429 @@
...
+
+### Cards to keep in mind
+
+#### Maindeck
+
+<row variant="hand"> {{!Orim's Chant}} {{!Veil of Summer}} {{!Echo of Eons}}</row>
+
+#### Sideboard
+
+<row variant="hand"> {{!Grapeshot}}</row>
```
Comment (Doishy) [inline markdown/articles/2021/08/22/Non-blue-fair-decks.md:234]: Unsure Grapeshot is the issue.....
Relay might be worth a mention (pre DD mostly).

File: markdown/articles/2021/08/22/Non-blue-fair-decks.md [hunk 12]
```diff
@@ -0,0 +1,429 @@
...
+
+#### Sideboard
+
+<row variant="hand"> {{!Collector Ouphe}} {{!Thoughtseize}}</row>
+
+### Gameplay
+
+#### Pre Doomsday
+
+Since Allosaurus Shepherd removed all interaction in this matchup, it's just racing and countering some Thoughtseize if necessary.
```
Comment (Doishy) [inline markdown/articles/2021/08/22/Non-blue-fair-decks.md:269]: Maybe discuss what are good targets for Dazes / Forces and what can be ignored?

File: markdown/articles/2021/08/22/Non-blue-fair-decks.md [hunk 13]
```diff
@@ -0,0 +1,429 @@
...
+Resolving Doomsday should not be too hard, Aluren plays 4 Force of Will main and 2 Force of Negation in the sideboard. That's often not enough when they also try to set up their own kill. In game 1 one counterspell is often enought to force your Doomsday through. In game 2 you want enough interaction to beat atleast 2 Force effects from their side and try to keep an eye out for Opposition Agent.
+
+#### Post Doomsday 
+
+Many lists play Endurance, like most green decks in the current meta, so prepare to play around atleast one copy.  
+If they fought about your Doomsday, they often don't have enough resources to go off, but that is not always the case, especially since Acererak the Archlich joined the format. Should they still be able to go off and you have enough cyclers in hand to cycle to Thassa's Oracle, be careful with casting it of Aluren. You have to cast it when their combo creature is on the stack, else they can react and combo off while the Thassa's Oracle trigger is still no the stack.
```
Comment (Doishy) [inline markdown/articles/2021/08/22/Non-blue-fair-decks.md:301]: If they fought about -> If they fought over

File: markdown/articles/2021/08/22/Non-blue-fair-decks.md [hunk 14]
```diff
@@ -0,0 +1,429 @@
...
+#### Sideboard
+
+<row variant="hand"> {{!Iona, Shield of Emeria}}</row>
+
+### Gameplay
+
+#### Pre Doomsday
+
+Do NOT keep hands without countermagic. You'll need it to protect your own combo and fighting off theirs. With all their discard effects one free counter is often not enough. The only reason to keep hands without protection are turn 1 kills.  
+When fighting their combo, try always to counter the Entomb effects, without a good target their reanimation spells are useless. And as a little sidenote, most lists play only 8 ways to creatures into the graveyard, but 12 reanimation spells.
```
Comment (Doishy) [inline markdown/articles/2021/08/22/Non-blue-fair-decks.md:322]: An excellent point to make.

File: markdown/articles/2021/08/22/Non-blue-fair-decks.md [hunk 15]
```diff
@@ -0,0 +1,429 @@
...
+
+#### Sideboard
+
+<row variant="hand"> {{!Iona, Shield of Emeria}}</row>
+
+### Gameplay
+
+#### Pre Doomsday
+
+Do NOT keep hands without countermagic. You'll need it to protect your own combo and fighting off theirs. With all their discard effects one free counter is often not enough. The only reason to keep hands without protection are turn 1 kills.
```
Comment (Doishy) [inline markdown/articles/2021/08/22/Non-blue-fair-decks.md:321]: Maybe rephrase. otp vs otd can be a big difference maker here so not sure it is as clear cut as "always have force" etc

File: markdown/articles/2021/08/22/Non-blue-fair-decks.md [hunk 16]
```diff
@@ -0,0 +1,429 @@
...
+### Gameplay
+
+#### Pre Doomsday
+
+Do NOT keep hands without countermagic. You'll need it to protect your own combo and fighting off theirs. With all their discard effects one free counter is often not enough. The only reason to keep hands without protection are turn 1 kills.  
+When fighting their combo, try always to counter the Entomb effects, without a good target their reanimation spells are useless. And as a little sidenote, most lists play only 8 ways to creatures into the graveyard, but 12 reanimation spells.
+
+#### Post Doomsday
+
+One of the scariest creatures they can reanimate post Doomsday is Ashen Rider, so try to be carefull with your lands and don't be stingy with your lotus petals.
```
Comment (Doishy) [inline markdown/articles/2021/08/22/Non-blue-fair-decks.md:326]: Not Iona?

File: markdown/articles/2021/08/22/Non-blue-fair-decks.md [hunk 17]
```diff
@@ -0,0 +1,429 @@
...
+
+## Sneak and Show
+
+### Gameplan
+
+### Cards to keep in mind
+
+#### Maindeck
+
+<row variant="hand"> {{!Daze}} {{!Force of Will}}</row>
```
Comment (Doishy) [inline markdown/articles/2021/08/22/Non-blue-fair-decks.md:336]: Add Sneak attack. The possibility of a hasty dude can sometimes be the kicker,

File: markdown/articles/2021/08/22/Non-blue-fair-decks.md [hunk 18]
```diff
@@ -0,0 +1,429 @@
...
+
+## Hogaak
+
+### Gameplan
+
+### Cards to keep in mind
+
+#### Maindeck
+
+<row variant="hand"> {{!Hedron Crab}} {{!Vengevine}} {{!Cabal Therapy}} {{!Altar of Dementia}}</row>
```
Comment (Doishy) [inline markdown/articles/2021/08/22/Non-blue-fair-decks.md:388]: Bloodghast. DD halving life total means these guys will always have haste on a ptt pile...

File: markdown/articles/2021/08/22/Non-blue-fair-decks.md [hunk 19]
```diff
@@ -0,0 +1,429 @@
...
+#### Pre Doomsday
+
+The Hogaak matchup can be pretty tricky, because they have two ways to kill you, either with damage or with mill. They don't even have to completely mill your 60 cards, the pure on board threat of milling us is often enough to make a kill with Doomsday impossible.  
+There are two iterations of Hogaak, a jund version and BUG one. The BUG versions is scarier by far, thanks to Hedron Crab.  
+Speed is the only way to beat Hogaak, they are just to explosive to give them time and you don't want to play against Hedron Crab and a uncracked Fetchlands.  
+They don't play any sideboard hate for Doomsday, but countermagic is still very important to keep them in check and protect your combo.
+
+#### Post Doomsday 
+
+Because Hogaak plays no interaction, you can build Brainstorm piles when ever you can. Should you have to pass the turn, make sure you have answers at your disposal.
```
Comment (Doishy) [inline markdown/articles/2021/08/22/Non-blue-fair-decks.md:403]: Change "no interaction". 
Crab, Altar, Cabal Therapy and Trophy all fit here.

File: markdown/articles/2021/08/22/Non-blue-fair-decks.md [hunk 20]
```diff
@@ -0,0 +1,429 @@
...
+
+#### Sideboard
+
+<row variant="hand"> {{!Mindbreak Trap}}</row>
+
+### Gameplay
+
+#### Pre Doomsday
+
+Similar to Hogaak it's a race of die matchup, but Madness has zero interaction. They just slam and hope it's enough, and we do the same.
```
Comment (Doishy) [inline markdown/articles/2021/08/22/Non-blue-fair-decks.md:423]: race of die -> race or die

File: markdown/articles/2021/08/22/Non-blue-fair-decks.md [hunk 21]
```diff
@@ -0,0 +1,429 @@
...
+
+#### Pre Doomsday
+
+Similar to Hogaak it's a race of die matchup, but Madness has zero interaction. They just slam and hope it's enough, and we do the same.  
+Lion's Eye Diamond is a very scary card in their deck, that leads to super explosive turns. It should be countered 90% of the time.  
+Should they have Mindbreak Trap in hand post sideboard, it's often pretty obvious. It's very hard for them to keep interaction up while playing out their threats.
+
+#### Post Doomsday
+
+Their only ways to put hasty threats onto the board are Vengevine and Anger. This makes it very easy to predict their damage and if it's safe to pass the turn.
```
Comment (Doishy) [inline markdown/articles/2021/08/22/Non-blue-fair-decks.md:429]: Maybe mention to check if they have a mountain or not re: Anger

File: markdown/articles/2021/08/22/Non-blue-fair-decks.md (hunk 9 above)
Comment (Lutri4eva) [reply markdown/articles/2021/08/22/Non-blue-fair-decks.md:124]: GW Depths has it's own section, but many current Maverick lists i found online had Depths as 2nd plan

File: markdown/articles/2021/08/22/Non-blue-fair-decks.md (hunk 11 above)
Comment (Lutri4eva) [reply markdown/articles/2021/08/22/Non-blue-fair-decks.md:234]: I'm unsure, but in my matches against TES Relay was irrelevant.
Grapeshot is a tool to kill you post Doomsday with less mana than Tendrils

File: markdown/articles/2021/08/22/Non-blue-fair-decks.md (hunk 13 above)
Comment (Lutri4eva) [reply markdown/articles/2021/08/22/Non-blue-fair-decks.md:301]: fixed

File: markdown/articles/2021/08/22/Non-blue-fair-decks.md (hunk 14 above)
Comment (Lutri4eva) [reply markdown/articles/2021/08/22/Non-blue-fair-decks.md:322]: fixed

File: markdown/articles/2021/08/22/Non-blue-fair-decks.md (hunk 19 above)
Comment (Lutri4eva) [reply markdown/articles/2021/08/22/Non-blue-fair-decks.md:403]: True.
Changed to no interaction on the stack

File: markdown/articles/2021/08/22/Non-blue-fair-decks.md (hunk 6 above)
Comment (Lutri4eva) [reply markdown/articles/2021/08/22/Non-blue-fair-decks.md:9]: fixed

File: markdown/articles/2021/08/22/Non-blue-fair-decks.md (hunk 7 above)
Comment (Lutri4eva) [reply markdown/articles/2021/08/22/Non-blue-fair-decks.md:10]: added

File: markdown/articles/2021/08/22/Non-blue-fair-decks.md (hunk 8 above)
Comment (Lutri4eva) [reply markdown/articles/2021/08/22/Non-blue-fair-decks.md:108]: fixed

File: markdown/articles/2021/08/22/Non-blue-fair-decks.md (hunk 18 above)
Comment (Lutri4eva) [reply markdown/articles/2021/08/22/Non-blue-fair-decks.md:388]: Added
Also added a small section about hasty damage post Doomsday

File: markdown/articles/2021/08/22/Non-blue-fair-decks.md (hunk 20 above)
Comment (Lutri4eva) [reply markdown/articles/2021/08/22/Non-blue-fair-decks.md:423]: fixed

File: markdown/articles/2021/08/22/Non-blue-fair-decks.md (hunk 21 above)
Comment (Lutri4eva) [reply markdown/articles/2021/08/22/Non-blue-fair-decks.md:429]: done

File: markdown/articles/2021/08/22/Non-blue-fair-decks.md (hunk 16 above)
Comment (Lutri4eva) [reply markdown/articles/2021/08/22/Non-blue-fair-decks.md:326]: Added a small part about Iona, but tbh if they reanimate Iona you are just dead.

File: markdown/articles/2021/08/22/Non-blue-fair-decks.md (hunk 17 above)
Comment (Lutri4eva) [reply markdown/articles/2021/08/22/Non-blue-fair-decks.md:336]: added

**Comment by Lutri4eva** [comment]
> For now depending on the final length I think either could work but I prefer a chapter, so that the route already exists and links made today will still work in the next version.

I think that's the best way to do it.

**Comment by angrybacon** [comment]
> @Lutri4eva The file should be Markdown and should be nested in a folder named after the date. Moreover, prefer lowercase _kebab-case_ over _snake_case_:
> 
> ```diff
> -markdown/articles/2021/08/Non_blue_fair_decks.txt
> +markdown/articles/2021/08/18/non-blue-fair-decks.md
> ```

I see you added the date sub-folder 👌 However I think you fell in Git's trap of case-insensitiveness. 

```diff
-markdown/articles/2021/08/18/Non-blue-fair-decks.md
+markdown/articles/2021/08/18/non-blue-fair-decks.md
```

No worries I'll deal with it on release. 😄 

For posterity, to have Git care about case-sensitive file changes, add the following to your Git configuration file(s). More details here: https://git-scm.com/book/en/v2/Getting-Started-First-Time-Git-Setup

```diff
 [user]
     ...
+[core]
+    ignorecase = false
```

**Comment by angrybacon** [comment]
> > For now depending on the final length I think either could work but I prefer a chapter, so that the route already exists and links made today will still work in the next version.
> 
> I think that's the best way to do it.

This means you no longer need the banner, and that can move the file into the chapters. You can inspect other chapters to see what the frontmatter -- the first few lines of metadata -- look like there. The URL will be forged and built automatically.

```diff
-markdown/articles/2021/08/18/Non-blue-fair-decks.md
+markdown/chapters/meandeck/matchups.md
```

File: markdown/articles/2021/08/22/Non-blue-fair-decks.md (hunk 9 above)
Comment (Doishy) [reply markdown/articles/2021/08/22/Non-blue-fair-decks.md:152]: I think maybe keep it out for now?

Comment (Lutri4eva) [reply markdown/articles/2021/08/22/Non-blue-fair-decks.md:152]: ok

**Comment by angrybacon** [comment]
@Lutri4eva Pushed a few changes to your branch directly. Fixed some formatting issues and spotted a couple typos, this was just a preliminary work so that we can review the whole document more easily (diffs will be more readable on shorter lines).

Hopefully I can release the new Wiki soon so that we can take advantage of collapsible content since the document really is huge 😅

**Comment by Lutri4eva** [comment]
> @Lutri4eva Pushed a few changes to your branch directly. Fixed some formatting issues and spotted a couple typos, this was just a preliminary work so that we can review the whole document more easily (diffs will be more readable on shorter lines).
> 
> Hopefully I can release the new Wiki soon so that we can take advantage of collapsible content since the document really is huge 😅

Nice, ty!

Yeah, it got pretty big. And it won't get smaller in the future.
